tests/integration/objectType_3.phpt

```diff
@@ -0,0 +1,26 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+
+v::objectType()->assert(new stdClass);
+v::objectType()->check(new stdClass);
+?>
+--EXPECTF--
+--FILE--
```

Comment:
Could you split it into 2 different tests?

I could do it here, but I'm not sure what is the correct one